src/android/app/src/main/java/io/github/lime3ds/android/features/settings/ui/viewholder/SwitchSettingViewHolder.kt

```diff
@@ -33,10 +37,20 @@ class SwitchSettingViewHolder(val binding: ListItemSettingSwitchBinding, adapter
             adapter.onBooleanClick(item, bindingAdapterPosition, binding.switchWidget.isChecked)
         }
 
-        binding.switchWidget.isEnabled = setting.isEditable
+        binding.switchWidget.isEnabled = if (setting.isEditable) {
+        isForceMaxGpuClocksClickable()
+        } else { 
+            setting.isEditable
+        }
+
         if (setting.isEditable) {
-            binding.textSettingName.alpha = 1f
-            binding.textSettingDescription.alpha = 1f
+            if (!isForceMaxGpuClocksClickable()) {
+                binding.textSettingName.alpha = 0.5f
+                binding.textSettingDescription.alpha = 0.5f
+            } else {
+                binding.textSettingName.alpha = 1f
+                binding.textSettingDescription.alpha = 1f
+            }
```

Comment:
Set a variable to either 0.5 or 1 depending on the condition, then set `binding.textSettingName.alpha` and `binding.textSettingDescription.alpha` to the value of that variable